From: Gert Wollny <gw.fossdev@gmail.com>
Date: Mon, 20 May 2019 12:51:17 +1000
Subject: Remove the FORCE from the use_system_glew=off option

The embedded version of GLEW seems to be older than the version
provided by Debian, so I assume that the fixes have been applied.
---
 Rendering/OpenGL2/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Rendering/OpenGL2/CMakeLists.txt b/Rendering/OpenGL2/CMakeLists.txt
index 479816e..2eca1f6 100644
--- a/Rendering/OpenGL2/CMakeLists.txt
+++ b/Rendering/OpenGL2/CMakeLists.txt
@@ -311,7 +311,7 @@ vtk_module_library(${vtk-module} ${Module_SRCS})
 vtk_opengl_link(${vtk-module})
 if(VTK_USE_X)
   vtk_module_link_libraries(${vtk-module}
-    LINK_PUBLIC ${X11_LIBRARIES} ${X11_Xt_LIB})
+    LINK_PUBLIC ${GLEW_LIBRARY} ${X11_LIBRARIES} ${X11_Xt_LIB})
 elseif(VTK_USE_COCOA)
   vtk_module_link_libraries(${vtk-module} LINK_PUBLIC "-framework Cocoa")
 endif()
